paperqa/utils.py

```diff
@@ -345,6 +346,24 @@ def remove_substrings(target: str, substr_removal_list: Collection[str]) -> str:
     return target
 
 
+def replace_acute_accents(text: str) -> str:
+    """Replaces acute accent with apostrophe."""
+
+    # Used for any client that returns names with acutes
+    def replace_acute(match):
+        return f"'{match.group(1)}"
+
+    nfd = unicodedata.normalize("NFD", text)
+    converted = re.sub(r"([aeiouAEIOU])\u0301", replace_acute, nfd)
+    return unicodedata.normalize("NFC", converted)
+
+
+def remove_acutes(text: str) -> str:
+    return "".join(
+        c for c in unicodedata.normalize("NFD", text) if unicodedata.category(c) != "Mn"
+    )
```

Comment:
What is the difference with `replace_acute_accents` and `remove_acutes`? It seems to be quite similar. Can you either update the name of `remove_acutes` or add a docstring to it to help clarify

Reply:
the authors in the other clients will have apostrophes instead of acutes
but their keys won't have either

the "replacement" function keeps the author format consistent
the "remove" function keeps the key format consistent

(this was caught by our awesome unit testing; `Lála` was the lastname that triggered this discovery)

Reply:
Okay I gotchu. I wonder if we can combine these two functions and have a `bool` flag to make it more intuitive? Defer to you here, at least maybe add a docstring

Reply:
combined!